resources/views/layouts/input.blade.php

```diff
@@ -1,5 +1,13 @@
-<div class="field">
-<div class="control">
-<input class="input is-rounded" type="text" placeholder="Rounded input">
-</div> 
-</div> 
+<article class="media">
+  <figure class="media-left">
+    <p class="image is-64x64">
```

Comment:
Minor but; is it possible to do without the `<p>`-tag? 
Maybe just add the image helper class to the `<img>` -tag :)